.github/workflows/label.yml

```
@@ -51,4 +52,78 @@ jobs:
                   removeLabel(label.name);
                 }
             }
-          }
\ No newline at end of file
+          }
+
+  add_label:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const I18N_FILES = [
+                "01-core",
+                "02-strings",
+                "03-dialogs",
+                "04-network",
+                "05-feedback",
+                "06-statistics",
+                "07-cardbrowser",
+                "08-widget",
+                "09-backup",
+                "10-preferences",
+                "11-arrays",
+                "16-multimedia-editor",
+                "17-model-manager",
+                "18-standard-models",
+                "marketdescription",
+                "ankidroid-titles",
+            ];
+
+            let stringsLabel = "strings";
+
+            async function addLabel(labels) {
+              await github.rest.issues.addLabels({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number,
+                labels: labels
+              });
+            }
+
+            async function addComments() {
+              await github.rest.issues.createComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number,
+                body: `Message to maintainers, this PR contains strings changes. So read more about updating strings on wiki,
+                      https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#localization-administration
+                      https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#download-localized-strings`
```

```suggestion
                body: `Message to maintainers, this PR contains strings changes. 
                1. Before merging this PR, it is best to run the "Sync Translations" github action,
                    then make and merge a PR from the i18n_sync branch to get translations cleaned out.
                2. Then merge this PR, and immediately do another translations PR so the huge change made
                     by this PR's key changes are all by themselves.
                
                Read more about updating strings on wiki,
                      https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#localization-administration
                      https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#download-localized-strings`
```

What do you think? Are double quotes allowed in the string? Probably but worth asking.

---

This message is better. Double quotes are allowed.
Thanks